Mathlib/Topology/Compactness/Compact.lean

```diff
@@ -612,6 +612,18 @@ theorem cocompact_eq_cofinite (X : Type*) [TopologicalSpace X] [DiscreteTopology
   simp only [cocompact, hasBasis_cofinite.eq_biInf, isCompact_iff_finite]
 #align filter.cocompact_eq_cofinite Filter.cocompact_eq_cofinite
 
+/-- A filter is disjoint from the cocompact filter if and only if it contains a compact set. -/
+theorem disjoint_cocompact_left (f : Filter X) :
```

Comment:
```suggestion
theorem cocompact.disjoint_iff_left (f : Filter X) :
```
I am not 100% sure, but I am following the naming of [Filter.HasBasis.disjoint_iff_left](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Order/Filter/Bases.html#Filter.HasBasis.disjoint_iff_left).

(Similarly below).

Reply:
I'm following the naming of [Filter.disjoint_cofinite_left](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Order/Filter/Cofinite.html#Filter.disjoint_cofinite_left).